conftest.py

```diff
@@ -17,4 +20,29 @@ def driver():
 def start_page(driver):
     """Creates start page object"""
     driver.get(BaseConstants.URL)
+    driver.implicity_wait(1)
     return StartPage(driver)
+
+
+@pytest.fixture()
+def hello_page(driver):
```

Comment:
It is of work like this. You cannot just create any page. Only start page can be created from driver by opening it, all other pages are have to be crated by navigating to them under the page objects

Reply:
Fixing is delete unnecessary wrong drivers at conftest.py